### PR TITLE
Fix forecast details in mobile view

### PR DESCRIPTION
**Pull request by BartlomiejWojcik**

### CODE REVIEW

File: src/app/forecast/hourly-forecast/hourly-forecast.component.ts [hunk 1]
```diff
@@ -162,4 +182,46 @@ export class HourlyForecastComponent {
...
+   * @param event onPage event
+   */
+  pageChanged(e: { page: number }): void {
+    const timeStamp =
+      this.detailsDate && typeof this.detailsDate !== 'number'
+        ? this.detailsDate.getTime()
+        : this.detailsDate;
+    if (this.isSmallScreen()) {
+      const index = this.getForecast().findIndex((x) => x.dt === timeStamp);
+      console.log(index);
```
Comment (ArturBa) [inline src/app/forecast/hourly-forecast/hourly-forecast.component.ts:217]: please remove console.log

**Review by ArturBa — CHANGES_REQUESTED** [review]
please remove console.log

File: src/app/forecast/hourly-forecast/hourly-forecast.component.ts (hunk 1 above)
Comment (BartlomiejWojcik) [reply src/app/forecast/hourly-forecast/hourly-forecast.component.ts]: Done

**Review by ArturBa — APPROVED** [review]
looks fine :D

**Pull request merged**